extensions/mssql/src/mssql.d.ts

```diff
@@ -1379,6 +1380,22 @@ declare module 'mssql' {
 			containmentTypes: string[];
 			recoveryModels: string[];
 		}
+
+		export interface PropertiesInfo extends SqlObject {
+			// Add common properties between server and database
```

Comment:
For common properties you should make a common base interface `ServerDatabaseBasePropertiesInfo` or something, then extend it for adding Server/DB specific properties

Reply:
As mentioned during the PR as well, I think given that server & database are two completely separate objects we shouldn't try to create any set of common properties and instead just define each of them separately (even if there's duplicates)

I'm not even sure what common properties aside from name there would be - the ones you have here are specific to servers AFAIK and aren't something that shows up in DB properties.